lib/cfg.c

```diff
@@ -543,6 +543,10 @@ _load_file_into_string(const gchar *fname)
 
   if (!g_file_get_contents(fname, &buff, NULL, NULL))
     {
+      msg_error("Error opening configuration file",
```

Comment:
Just check the glib documentation, and for this function does not mention `errno` (in case of some function it says that e `errno` is going to be set), so I agree on using error.